astra/src/main/java/com/slack/astra/metadata/core/AstraMetadataStore.java

```diff
@@ -327,6 +347,24 @@ public boolean hasSync(String path) {
     }
   }
 
+  /**
+   * Checks if an ETCD node exists synchronously.
+   *
+   * @param path the path to check
+   * @return true if the node exists, false otherwise.
+   * @throws RuntimeException from EtcdMetadataStore if a connection error or other unrecoverable
+   *     error occurs. Note that the node not existing is NOT treated as an error; in that case, the
+   *     method returns false.
+   */
+  public boolean hasEtcdOnlySync(String path) {
+    boolean existsInEtcd = etcdStore.hasSync(path);
```

Comment:
Kinda wonky not just returning the boolean, but this is short-lived code. 🤷

Reply:
Wow, I think I was copying from the other method and paring it down and missed this completely. I'll just return the boolean..